components/pages/home/MemberCard.js

```diff
@@ -0,0 +1,50 @@
+import Image from "next/image";
+
+const MemberCard = ({ login, name, html_url, avatar_url }) => {
+  const dummyImage =
+    "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAFAAAABQCAYAAACOEfKtAAAAAXNSR0IArs4c6QAAAPtJREFUeF7t0rENAAAMwrDy/9M9wmvYs1jsGgmM6uILEE8QYIAogHkPDBAFMO+BAaIA5j0wQBTAvAcGiAKY98AAUQDzHhggCmDeAwNEAcx7YIAogHkPDBAFMO+BAaIA5j0wQBTAvAcGiAKY98AAUQDzHhggCmDeAwNEAcx7YIAogHkPDBAFMO+BAaIA5j0wQBTAvAcGiAKY98AAUQDzHhggCmDeAwNEAcx7YIAogHkPDBAFMO+BAaIA5j0wQBTAvAcGiAKY98AAUQDzHhggCmDeAwNEAcx7YIAogHkPDBAFMO+BAaIA5j0wQBTAvAcGiAKY98AAUQDzHoiAD0PoAFE/HRk+AAAAAElFTkSuQmCC";
+  return (
+    <div className="team-card m-5">
+      <div
+        className="mx-auto relative max-w-max tooltip top"
+        pesan={name === null ? login : name}
+      >
+        <div className="relative flex justify-center items-center rounded-full overflow-hidden w-20 h-20 shadow-md">
+          <Image
+            src={avatar_url ? `${avatar_url}&s=80` : dummyImage}
```

Comment:
Lanjutan dari yang review di atas, url avatarnya aja yang dimasukin

```suggestion
            src={`${avatar_url}&s=80`}
```

Tapi sebenernya kalo `next/image` udah ngeresize kan ?

Reply:
Saya pakai `layout="fill"`, jadi ukurannya ngikutin ukuran parrentnya.